src/fmu/dataio/_filedata_provider.py

```diff
@@ -153,13 +155,13 @@ def _get_filestem(self):
             stem = stem.replace("__", "_")
 
         # treat norwegian special letters
+        # BUG(?): What about germen letter like "Ü"?
```

Comment:
Good point, plus characters used in the Portuguese alphabet for Brazil

Reply:
I looked a bit at https://pypi.org/project/Unidecode/ but it not a dropin replacement (have not checked the docs if there any options we could use)